### ISSUE TITLE
Flex and Foundations: add alt text for accessibility.

### ISSUE THREAD

**Issue by Amandasilvbr** [TheOdinProject/css-exercises#201]
Complete the following REQUIRED checkboxes:
-   [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [X] The title of this issue follows the `location for request: brief description of request` format, e.g. `Foundations exercises: Add exercise for XYZ`

The following checkbox is OPTIONAL:

-   [X] I would like to be assigned this issue to work on it

<hr>

**1. Description of the Feature Request:**

Four html files were missing the alt attribute in the img tag. This compromises accessibility. In addition, the use of alternative texts is considered a good practice and should be encouraged whenever appropriate.

**2. Acceptance Criteria:**

- [x] Add alternative texts where missing
